### ISSUE TITLE
SLF4J warning (2024)

### ISSUE THREAD

**Issue by raquo** [foundweekends/giter8#931]
## steps

Steps to reproduce:

```
sbt new raquo/scalajs.g8
```

sbt or Giter8 version:

```
sbt --version
sbt version in this project: 1.10.2
sbt script version: 1.10.2
```

Not sure how to tell giter8 version that's installed with sbt.

## problem

`sbt new` outputs SLF4J warnings:

```
sbt new raquo/scalajs.g8
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
SLF4J: See http://www.slf4j.org/codes.html#StaticLoggerBinder for further details.
```

## expectation

No warnings

## notes

I see this very old issue about the same problem, which was fixed back then: https://github.com/foundweekends/giter8/issues/211

**Comment by eed3si9n** [comment]
It seems like I removed logback in 2019 (https://github.com/foundweekends/giter8/pull/444) when I implemented giter8-launcher mechanism that dynamically chooses Giter8 version to fix #435 "SLF4J: Class path contains multiple SLF4J bindings" in the issue I speculate that it's because JGit is pulling in slf4j-log4j12. So whatever the version of JGit Giter8 using now probably stop including slf4j binding. We can re-add a binding.
